src/front/ticketer/assets/.env

```diff
@@ -0,0 +1 @@
+BACKEND_URL=localhost:5166
```

Comment:
There is a problem with this implementation of dotenv. For now it is okay solution but in future we might want to consider other alternatives